packages/sanity/src/core/studio/components/navbar/free-trial/DescriptionSerializer.tsx

```diff
@@ -0,0 +1,133 @@
+import {PortableText, PortableTextComponents} from '@portabletext/react'
+import {LinkIcon} from '@sanity/icons'
+import {PortableTextBlock} from '@sanity/types'
+import {Box, Card, Flex, Text} from '@sanity/ui'
+import styled from 'styled-components'
+import React, {useEffect, useState} from 'react'
+
+interface DescriptionSerializerProps {
+  blocks: PortableTextBlock[]
+}
+
+const Divider = styled(Box)`
+  height: 1px;
+  background: var(--card-border-color);
+  width: 100%;
+`
+
+const SerializerContainer = styled.div`
+  // Remove margin bottom to last box.
+  > [data-ui='Box']:last-child {
+    margin-bottom: 0;
+  }
+`
+
+const Link = styled.a<{useTextColor: boolean}>`
+  font-weight: 600;
+  color: ${(props) => (props.useTextColor ? 'var(--card-muted-fg-color) !important' : '')};
+`
+
+const DynamicIconContainer = styled.span`
+  > svg {
+    display: inline;
+    font-size: calc(21 / 16 * 1rem) !important;
+    margin: -0.375rem 0 !important;
+    *[stroke] {
+      stroke: currentColor;
+    }
+  }
+`
+const DynamicIcon = (props: {icon: {url: string}}) => {
+  const [ref, setRef] = useState<HTMLSpanElement | null>(null)
+  useEffect(() => {
+    if (!ref) return
+    fetch(props.icon.url)
```

Comment:
It might be a good idea to check the response status here (`response.ok`). Fetch doesn't throw an error for non-network errors, so e.g. if the icon cannot be found (`404`), the text content of the `404` page would be injected into the icon container. The same thing would occur for `5xx` server errors, etc.